app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
```

Comment:
Not needed.
I suggest you avoid using `session`

Reply:
removed!